src/components/playbackSettings/playbackSettings.js

```diff
@@ -190,6 +190,11 @@ function loadForm(context, user, userSettings, systemInfo, apiClient) {
         context.querySelector('.fldEnableHi10p').classList.remove('hide');
     }
 
+    //Show hls segment length setting for Web0s only, as the setting only aims to fix an issue on that platform.
+    if (browser.isWeb0s) {
```

Comment:
Use `web0s` property - that function is not exported.
```suggestion
    if (browser.web0s) {
```

Reply:
Good catch